File: packages/patrol/android/src/main/kotlin/pl/leancode/patrol/Logger.kt
package pl.leancode.patrol

import android.util.Log

object Logger {
    private const val TAG = "PatrolServer"

    fun e(msg: String) {
        Log.e(TAG, msg)
    }

    fun e(msg: String, tr: Throwable?) {
        Log.e(TAG, msg, tr)
    }

    fun i(msg: String) {
        Log.i(TAG, msg)
    }

    fun d(msg: String) {
        Log.d(TAG, msg)
    }
}

File: packages/patrol/darwin/Classes/AutomatorServer/Logger.swift
import Foundation

class Logger {
  private let TAG = "PatrolServer"

  private init() {}

  static let shared = Logger()

  func d(_ msg: String) {
    NSLog("\(TAG): DEBUG: \(msg)")
  }

  func i(_ msg: String) {
    NSLog("\(TAG): INFO: \(msg)")
  }

  func e(_ msg: String) {
    NSLog("\(TAG): ERROR: \(msg)")
  }
}
